.env.example

```
@@ -0,0 +1,72 @@
+
+# ========================================
+# DEPLOYMENT (Production Only)
+# ========================================
+# Port override (usually set by hosting provider)
+PORT=8080
+
+# Railway specific
+# RAILWAY_PUBLIC_DOMAIN=your-app.railway.app
+
+# Vercel specific  
+# REACT_APP_API_URL=https://your-backend.railway.app
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)